docs/handbook/squeak.md

```diff
@@ -0,0 +1,165 @@
+<!--
+
+The contents of this Documentation are subject to the Public Documentation License Version 1.01
+ (the "License"); you may only use this Documentation if you comply with the terms of this License.
+A copy of the License is available at http://illumos.org/license/PDL.
+
+
+The Original Documentation is _________________.
+
+The Initial Writer of the Original Documentation is ___________ Copyright (C)_________[Insert year(s)].
+All Rights Reserved. (Initial Writer contact(s):________________[Insert hyperlink/alias]).
+
+Contributor(s): ______________________________________.
```

Comment:
Please, fill in the copyright header.

Reply:
Feel free to enter my name (David Stes) and the year 2020.   I hope the squeak.md page is accepted.   It's currently not clear whether it will be accepted even if I fill out my name in the header.